scripts/deployL2-it.js

```diff
@@ -52,7 +52,7 @@ async function deployContract() {
   console.log("storage impl address is ", impl);
 
   const data = implContract.interface.encodeFunctionData("initialize", [
-    1572864, // minimumDiff 0.1 * 180 (3 minutes) * 1024 * 1024 / 12 = 1572864 for 0.1 replicas that can have 1M IOs in one epoch
+    5242880, // minimumDiff 0.1 * 600 (10 minutes) * 1024 * 1024 / 12 = 5242880 for 0.1 replicas that can have 1M IOs in one epoch
```

Comment:
Is it enough to solve the difficulty issue?

Reply:
the test passes after this change
https://github.com/ethstorage/es-node/actions/runs/11231405326/job/31335893875